### PR TITLE
Migration to drop channel_id column from PlaylistSnapshot

### PR DESCRIPTION
**Pull request by hebron-george**
https://github.com/happy-software/happy_youtube_watcher/issues/28

### CODE REVIEW

File: db/migrate/20210611014548_drop_channel_id_from_playlist_snapshot.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class DropChannelIdFromPlaylistSnapshot < ActiveRecord::Migration[6.1]
+  def change
+    remove_column :playlist_snapshots, :channel_id
```
Comment (hectron) [inline db/migrate/20210611014548_drop_channel_id_from_playlist_snapshot.rb:3]: Does this column currently have data?

**Review by hectron — COMMENTED** [review]
Based on the issue description rabbit-hole, it sounds like this data might need to be moved to another database. Is that the case?

Otherwise, this looks good to me!

File: db/migrate/20210611014548_drop_channel_id_from_playlist_snapshot.rb (hunk 1 above)
Comment (hebron-george) [reply db/migrate/20210611014548_drop_channel_id_from_playlist_snapshot.rb:3]: Yeah but I migrated it out to the `tracked_playlists` table already.

**Review by hectron — APPROVED** [review]
let's goooo

**Comment by hebron-george** [comment]
> Based on the issue description rabbit-hole, it sounds like this data might need to be moved to another database. Is that the case?
> 
> Otherwise, this looks good to me!

I'm not entirely sure where it should live tbh. This entire repo was basically born from a simple script so the data model wasn't really thought out.

Basically we have these giant snapshots of every (video + description + video metadata) for each playlist in `PlaylistSnapshot`.

We also have a concept of `TrackedPlaylist`s. So the idea is we iterate through each tracked playlist and take a daily playlist snapshot.

The `PlaylistSnapshot` existed first so it used to have the `channel_id` stored in a column on that table. I've since added that column to `TrackedPlaylist` and moved it there.

**Pull request merged**